Structure_Engine/Query/MaterialComposition.cs

```diff
@@ -136,6 +147,59 @@ public static MaterialComposition MaterialComposition(this ConcreteSection secti
         }
 
         /***************************************************/
+        /**** Public Methods - Interface                ****/
+        /***************************************************/
+
+        [PreviousVersion("4.3", "BH.Engine.Structure.Query.MaterialComposition(BH.oM.Structure.SectionProperties.ISectionProperty)")]
```

Comment:
```suggestion
        [PreviousVersion("5.0", "BH.Engine.Structure.Query.MaterialComposition(BH.oM.Structure.SectionProperties.ISectionProperty)")]
```
Versioning doesn't work - does this need to be made 5.0 because of the recent release?